Ports/.port_include.sh

```diff
@@ -797,36 +801,30 @@ do_dev() {
                 }
             done
         fi
+
+        git tag original
+
         popd
     )
 
-    [ -d "$git_repo" ] && [ ! -d "$workdir" ] && {
-        git clone --config core.autocrlf=false "$git_repo" "$workdir"
-    }
-
     [ -d "$workdir/.git" ] || {
-        >&2 echo "$workdir does not appear to be a git repository, if you did this manually, you're on your own"
-        if prompt_yes_no "Otherwise, press 'y' to remove that directory and clone it again"; then
-            rm -fr "$workdir"
-            git clone --config core.autocrlf=false "$git_repo" "$workdir"
-        else
-            exit 1
-        fi
+        >&2 echo "$workdir does not appear to be a git repository."
+        >&2 echo "If you want to use './package.sh dev', remove the directory manually and run the script again."
```

Comment:
`$workdir` is relative so this message might not be clear enough for the user. Maybe either report an absolute path or refer to `./package.sh clean` instead of asking them to remove it manually?

Reply:
> [...] or refer to ./package.sh clean instead of asking them to remove it manually

Good thing that we can now assume that `clean` always removes the entire working directory. Fixed! :^)